.github/workflows/coverage.yml

```diff
@@ -15,7 +15,7 @@ jobs:
   coverageReport:
     strategy:
       matrix:
-        api-level: [21, 21] 
+        api-level: [21, 27, 27, 21]
```

Comment:
revert this, useful to have to verify it is working but we don't want to commit this change